kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -469,21 +478,11 @@ jq --arg SSH_PUBLICKEY "${SSH_PUBLICKEY}" '.properties.linuxProfile.ssh.publicKe
 
 validate_and_restore_cluster_definition $AZURESTACK_CONFIGURATION_TEMP $AZURESTACK_CONFIGURATION || exit $ERR_API_MODEL
 
-if [ "$STORAGE_PROFILE" == "blobdisk" ]; then
-    log_level -w "Using blob disks requires AvailabilitySet, overriding availabilityProfile and storageProfile."
-    
-    cat $AZURESTACK_CONFIGURATION | \
-    jq --arg AvailabilitySet "AvailabilitySet" '.properties.agentPoolProfiles[0].availabilityProfile=$AvailabilitySet' | \
-    jq --arg StorageAccount "StorageAccount" '.properties.agentPoolProfiles[0].storageProfile=$StorageAccount' | \
-    jq --arg StorageAccount "StorageAccount" '.properties.masterProfile.storageProfile=$StorageAccount' \
-    > $AZURESTACK_CONFIGURATION_TEMP
-    
-    validate_and_restore_cluster_definition $AZURESTACK_CONFIGURATION_TEMP $AZURESTACK_CONFIGURATION || exit $ERR_API_MODEL
-fi
-
 if [ $IDENTITY_SYSTEM == "ADFS" ]; then
     log_level -i "Setting ADFS specific cluster definition properties."
     ADFS="adfs"
+    IDENTITY_SYSTEM_LOWER=$ADFS
```

Comment:
`IDENTITY_SYSTEM_LOWER` does not seem to be required, you already have `ADFS`.

Reply:
aks-engine needs the identity-system to be specified all in lower case. So AAD its  'azure_ad'  and for ADFS its "adfs"

Reply:
I get that, but both `ADFS` and `IDENTITY_SYSTEM_LOWER` have the same value.